pages/common/swig.md

```diff
@@ -0,0 +1,20 @@
+# swig
+
+> Generate bindings between C/C++ code and various high level languages (Go, Java, Javascript, Perl, PHP, Python, Tcl, Ruby, C#, D).
```

Comment:
This and the next line need to be rewritten. They contain too much information for someone who just wants to get a quick info about this program.

The rest is all well done. :+1:

Reply:
Again @zdroid, offering a solution is probably a good idea. For example:
```suggestion
> Generate bindings between C / C++ code and various high level languages such as Javascript, Python, C#, and more.
```